tests/myPlugin.cpp

```diff
@@ -108,7 +108,8 @@ class MyPlugin : public brayns::ExtensionPlugin
 
         // test properties from custom renderer
         brayns::PropertyMap props;
-        props.setProperty({"awesome", "Best property", 42, {0, 50}});
+        props.setProperty({"awesome", "Best property", int32_t(42),
+                           std::pair<int32_t, int32_t>{0, 50}});
```

Comment:
that looks a regression to me if it can no longer be declared like before. Why are the explicit types needed?

Reply:
They are not actually needed, fixed.